test/Dialect/FIRRTL/SFCTests/data-taps.fir

```diff
@@ -3,7 +3,10 @@
 ; - github.com/sifive/$internal:
 ;   - src/test/scala/grandcentral/DataTapsTest.scala
 
-circuit Top :
+circuit Top :%[[{
+   "class":"sifive.enterprise.firrtl.MarkDUTAnnotation" ,
+    "target":"~Top|Child"
+     }]]
```

Comment:
Nit: The whitespace and alignment are slightly bothering me. If this is legal, can we format it like this?

```suggestion
circuit Top : %[[{
    "class": "sifive.enterprise.firrtl.MarkDUTAnnotation",
    "target": "~Top|Child"
  }]]
```

I can be pretty flexible, but mostly just want the keys to line up :)

Same comment applies to several of the other tests where you had to add these annotations.